core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -76,6 +93,16 @@ oppia.directive('collectionNodeCreator', [
             );
           };
 
+          var convertTypeAheadToExplorationId = function(newExplorationId) {
```

Comment:
Typeahead seems to be one word: https://angular-ui.github.io/bootstrap/

Also, is the arg misnamed? Should it be "typeahead" or "typeaheadLabel" or "typeaheadOption" (or whatever it's typically called)?